app/menu.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-env node */
+const {app, Menu} = require("electron")
+const isDev = require("electron-is-dev")
+
+const defaultTemplate = [
+  {
+    label: "Edit",
+    submenu: [
+      {role: "undo"},
+      {role: "redo"},
+      {type: "separator"},
+      {role: "cut"},
+      {role: "copy"},
+      {role: "paste"},
+      {role: "delete"},
+      {role: "selectall"}
+    ]
+  },
+  {
+    role: "window",
+    submenu: [
+      {role: "minimize"},
+      {role: "close"}
+    ]
+  },
+  {
+    role: "help",
+    submenu: [
+      {
+        label: "Learn More",
+        click () { require("electron").shell.openExternal("https://www.classroom.github.com/assistant") }
```

Comment:
Does Classroom run locally?  It may be good to allow the host to be configurable. i.e.  `classroom.localhost`/assistant

Reply:
Yes, Classroom does run locally, but for now, I have pointed everything in the app to point towards `www.classroom.github.com` in production since we need to know if we're in localhost vs. production when we're fetching data from the Classroom API as well.

I presume a good solution for this would be something like setting an env variable towards a certain host when running locally vs. production in the future?